### PR TITLE
Update build steps for Development

### PR DESCRIPTION
**Pull request by 0xh3rman**
After `make bootstrap`, run `make unsigned-apk`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,6 +29,9 @@ build-test:
 test:
 	./gradlew connectedAndroidTest
 
+unsigned-apk:
```
Comment (gemcoder21) [inline Makefile:32]: Need to assemble release APK

Comment (0xh3rman) [reply Makefile:32]: There is already a release target below

**Comment by 0xh3rman** [comment]
@gemcoder21 changed pr title, to build reproducible build, needs more work to be done first
- built on Ubuntu
- different build variant and flavors
cc @furenster

**Comment by 0xh3rman** [comment]
Added https://github.com/gemwalletcom/gem-android/issues/7

**Pull request merged**